plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -776,7 +776,8 @@ fu_thunderbolt_device_write_firmware (FuDevice *device,
 	}
 
 	/* whether to wait for a device replug or not */
-	if (!fu_device_has_flag (device, FWUPD_DEVICE_FLAG_USABLE_DURING_UPDATE)) {
+	if (self->device_type != FU_THUNDERBOLT_DEVICE_TYPE_RETIMER &&
```

Comment:
I am not entirely sure this is always the case. This might be true only on the NDA case.

Reply:
OK I'll take just the other one until we can confirm what happens in device attached case.